src/main/java/seedu/address/ui/StatusBarFooter.java

```diff
@@ -40,12 +40,15 @@
     private StatusBar syncStatus;
     @FXML
     private StatusBar saveLocationStatus;
+    @FXML
+    private StatusBar totalPersons;
 
 
-    public StatusBarFooter(String saveLocation) {
+    public StatusBarFooter(String saveLocation, int totalPersons) {
```

Comment:
can use a set method, so you dont need to change theconstructor

Reply:
Already implemented a setTotalPersons method, and it is used in the constructor.
The constructor needs to take in an integer totalPersons to set the number in the status bar.